Microsoft.Azure.Cosmos.Encryption.Custom/src/EncryptionContainer.cs

```diff
@@ -1010,6 +1010,16 @@ internal sealed class EncryptionContainer : Container
             return this.ResponseFactory.CreateItemFeedResponse<T>(responseMessage);
         }
 
+#if PREVIEW
```

Comment:
This is the encryption project. It builds using both a project reference and the public nuget. You need to use the following flag to get it to build correctly. The entire encryption project is in preview. 

Example: https://github.com/Azure/azure-cosmos-dotnet-v3/pull/2935
```suggestion
#if SDKPROJECTREF
```

Reply:
made the fix